kolibri-electron/package.json

```diff
@@ -57,7 +57,7 @@
     "@electron-forge/maker-rpm": "^6.0.0",
     "@electron-forge/maker-squirrel": "^6.0.0",
     "@electron-forge/maker-zip": "^6.0.0",
-    "electron": "22.3.7"
+    "electron": "22.3.16"
```

Comment:
Sorry for the late review here. This version specifier looks weird to me. Is there a reason to specify a specific version, or could we do `~22.3.7` (for the latest patch release starting with `22.3.`) or `^22.3.7` (for the latest minor release starting with `22.`)? With that in place, `yarn upgrade` should do the right thing and update the lock file automatically.

Reply:
Sounds good!